samples/ClientCredentialsFlow/AuthorizationServer/AuthorizationServer.csproj

```diff
@@ -17,6 +17,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.1" />
```

Comment:
Ideally, we should use something like `$(EntityFrameworkCoreVersion)` so that we don't have to update all the .csproj when bumping versions.